README.md

```diff
@@ -197,6 +197,30 @@ describe Person do
 end
 ```
 
+### RSpec-Expectations
+
+Use this instead of the RSpec ifself in case you just use the specific module gem
```

Comment:
I think the English can be improved here, I was a little confused by it. I suggest:

```
`shoulda-matchers` can be configured to support using _only_ the `rspec-expectations` gem instead of the full RSpec suite.

Firstly, include `shoulda-matchers` in your Gemfile:
```

...it was much harder than I expected to find a good starting point here. Could you perhaps give a one-liner for why you want to use _only_ RSpec-Expectations and not the full suite? Speed? Load-time? Fewer dependencies?

Reply:
usage with only cucumber )